app/Http/Controllers/API/v1/Student/StudentActivitiesController.php

```diff
@@ -21,4 +24,20 @@ public function studentActivities(Request $request)
        return $this->successResponse(['data' => $activities], 200);
 
     }
+
+    public function followedActivities(Request $request)
+    {
+        $followedUsers = DB::table('user_follower')
```

Comment:
Please use the UserFollower Model and refrain from using db:table unless needed..

You can refer to the function above on how to access it.